src/components/media-loader.js

```diff
@@ -485,6 +485,10 @@ AFRAME.registerComponent("media-pager", {
     this.toolbar = null;
     this.onNext = this.onNext.bind(this);
     this.onPrev = this.onPrev.bind(this);
+    this.onSnap = this.onSnap.bind(this);
+    this.localSnapCount = 0;
+    this.isSnapping = false;
+    this.onSnapImageLoaded = () => (this.isSnapping = false);
```

Comment:
This never seems to be called and is the only thing mucking with `isSnapping` so they can both probably go

Reply:
oops, thats what i get for copy pasting